### ISSUE TITLE
Add vital sign action responsivness

### ISSUE THREAD

**Issue by PeterKuralt** [iryonetwork/wwm#144]
When choosing add vital sign from the Waiting list menu application does not respond. Input mask appears on the second click.

**Comment by mateuszkrasucki** [comment]
I doubt description is really accurate. Could we check it together?

**Comment by PeterKuralt** [comment]
You are right, the problem is that _add vital sign_ action is triggered only if user clicks on the text. The action should be triggered if user clicks on the field area as well.

**Comment by mateuszkrasucki** [comment]
@PeterKuralt I've changed the issue name. 
One more thing, I guess we're not aiming for now at making whole menu look like in the design?
